test/extended/util/test.go

```diff
@@ -215,7 +215,10 @@ func AnnotateTestSuite() {
 		if !excludedTestsFilter.MatchString(name) {
 			isSerial := strings.Contains(name, "[Serial]")
 			isConformance := strings.Contains(name, "[Conformance]")
+			isCSI := strings.Contains(name, "[CSI]")
```

Comment:
This is redundant (or should be), you should be doing this on like 485 (replace [CSI] with `[Suite:openshift/csi]`

Reply:
I believe that if I remove `isCSI`, CSI tests will be added also to `Suite:openshift/conformance/parallel` in the switch below. I still need `isCSI` not to do that.

I'll remove marking the test with `[CSI]`, I agree it's redundant.

Reply:
Fixed with `hasSuite` to catch any test that has already been categorized for OpenShift.